runtime.go

```diff
@@ -7,6 +7,21 @@ import (
 	"github.com/gates/gates/syntax"
 )
 
+type ToNativeOption int
+
+const (
+	OptionsDisableCircular ToNativeOption = iota
```

Comment:
`SkipCircularReference` is simpler and more accurate than `OptionsDisableCircular` since the behavior is returning a `nil` instead of an error when circular reference detected.